src/main/twirl/gitbucket/core/pulls/mergeguide.scala.html

```diff
@@ -77,23 +77,45 @@
           } else {
             <div class="merge-indicator merge-indicator-success"><span class="octicon octicon-check"></span></div>
             @if(status.hasMergePermission){
-              <span class="strong">Merging can be performed automatically.</span>
-              <div class="small">
-                Merging can be performed automatically.
-              </div>
+              @if(!pullreq.isDraft){
+                  <span class="strong">Merging can be performed automatically.</span>
+                  <div class="small">
+                  Merging can be performed automatically.
+                 </div>
+               } else {
```

Comment:
This part looks almost the same as below part. I think this code duplication can be avoided by devising conditional branching.

Reply:
ok, will avoid the duplication